mockbuild_test/stratisd.spec

```diff
@@ -94,16 +104,21 @@ tar --strip-components=1 --extract --verbose --file %{SOURCE2}
 %{__cargo} build %{?_smp_mflags} --release --bin=stratis-min --bin=stratisd-min --bin=stratis-utils --no-default-features --features engine,min,systemd_compat
 %{__cargo} rustc %{?_smp_mflags} --release --bin=stratis-str-cmp --no-default-features --features udev_scripts -- -Ctarget-feature=+crt-static
 %{__cargo} rustc %{?_smp_mflags} --release --bin=stratis-base32-decode --no-default-features --features udev_scripts -- -Ctarget-feature=+crt-static
+%{__cargo} build %{?_smp_mflags} --release --bin=stratis-dumpmetadata --no-default-features --features engine,extras,min
 %else
 %{__cargo} build %{?__cargo_common_opts} --release --bin=stratisd
 %{__cargo} build %{?__cargo_common_opts} --release --bin=stratis-min --bin=stratisd-min --bin=stratis-utils --no-default-features --features engine,min,systemd_compat
 %{__cargo} rustc %{?__cargo_common_opts} --release --bin=stratis-str-cmp --no-default-features --features udev_scripts -- -Ctarget-feature=+crt-static
 %{__cargo} rustc %{?__cargo_common_opts} --release --bin=stratis-base32-decode --no-default-features --features udev_scripts -- -Ctarget-feature=+crt-static
+%{__cargo} build %{?__cargo_common_opts} --release --bin=stratis-dumpmetadata --no-default-features --features engine,extras,min
 %endif
 a2x -f manpage docs/stratisd.txt
+a2x -f manpage docs/stratis-dumpmetadata.txt
 
 %install
 %make_install DRACUTDIR=%{dracutdir} PROFILEDIR=release
+%{__install} -Dpm0644 -t %{buildroot}%{_mandir}/man8 docs/stratis-dumpmetadata.8
```

Comment:
Drop these two install lines. They should be redundant now w/ the changes in PR https://github.com/stratis-storage/stratisd/pull/3258 . (They existed previously because we were thinking we wanted to release in Fedora but not RHEL and so the Makefile install target didn't install these bits. Now, it does.)

Reply:
OK; tested with stratisd PR 3258; that now must be merged before this one.